### PR TITLE
CSPL-1777: Installing UBI security patch docker

### PR DESCRIPTION
**Pull request by vivekr-splunk**
ubi security patch to docker images

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -30,6 +30,7 @@ ENV OPERATOR=/manager \
 
 RUN yum -y install shadow-utils
 RUN useradd -ms /bin/bash nonroot -u 1001
+RUN yum -y update-minimal --security --sec-severity=Important --sec-severity=Critical
```
Comment (akondur) [inline Dockerfile:33]: Hi @vivekr-splunk , would you mind explaining the importance of update-minimal here or in the description?

Comment (vivekr-splunk) [reply Dockerfile:33]: I removed minimal as i was testing installing shadow-utils which i was unable to do it , moved to latest that worked. we can move back if needed

**Review by akondur — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (vivekr-splunk) [reply Dockerfile:33]: to only install security errata use `yum update-minimal`

**Review by sgontla — APPROVED** [review]

**Review by jryb — APPROVED** [review]

**Review by kashok-splunk — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (smohan-splunk) [inline Dockerfile:33]: How can we confirm if the security patch has resolved the vulnerability? Or can this be done only during release time?

**Pull request merged**